.github/workflows/stress.yml

```diff
@@ -54,11 +54,10 @@ jobs:
       - run: mkdir -p $TEMPORAL_TESTING_LOG_DIR/tails
       - run: mkdir -p $TEMPORAL_TESTING_MEM_LOG_DIR
       - run: 'timeout ${{ inputs.test-timeout-minutes }}m npm run ${{ inputs.test-type }}'
-      # Tail the logs and upload as artifact on cancel or failure
       - run: for f in $TEMPORAL_TESTING_LOG_DIR/*.log; do tail -20000 $f > $TEMPORAL_TESTING_LOG_DIR/tails/$(basename $f); done
-        if: ${{ cancelled() || failure() }}
+        if: ${{ always() }}
       - uses: actions/upload-artifact@v3
-        if: ${{ cancelled() || failure() }}
+        if: ${{ always() }}
```

Comment:
Assuming this is temporary

Reply:
This is to have green output to compare to. Can compare to local output, but may be different due to platform or length?

Reply:
Should be similar to local, I guess it wouldn't hurt keeping